pkg/steps/release/import_release.go

```diff
@@ -219,6 +219,10 @@ func (s *importReleaseStep) run(ctx context.Context) error {
 	commands := fmt.Sprintf(`
 set -euo pipefail
 export HOME=/tmp
+mkdir -p $HOME/.docker
+if [[ -f /pull ]]; then
+	cp /pull $HOME/.docker/config.json
```

Comment:
must be `[[ -d` and the source location is wrong, I opened https://github.com/openshift/ci-tools/pull/1567/files and think its correct there
/hold